ChangeColorViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ChangeColorViewController.swift
+//  CheckMyPixel
+//
+//  Created by Stanislav Cherkasov on 17.01.18.
+//  Copyright © 2018 Stanislav Cherkasov. All rights reserved.
+//
+
+import UIKit
+
+class ChangeColorViewController: UIViewController {
+    
+    let backgroundColors = [UIColor.black, UIColor.red, UIColor.green, UIColor.blue, UIColor.white]
+    
+    var index = 0
+
+    @IBAction func changeColorButton(_ sender: Any) {
+       self.view.backgroundColor = backgroundColors[index]
+        
+        if index == backgroundColors.count - 1 {
```

Comment:
substitue this with a following line
`index = (index + 1) % backgroundColors.count`

Reply:
Explain here why this line is the same as if index == ... else statements